src/app/api/check-username-unique/route.ts

```diff
@@ -1,38 +1,40 @@
-import dbConnect from "@/lib/dbConnect";
-import UserModel from "@/model/User";
+
 import { z } from "zod";
 import { usernameValidation } from "@/schemas/signUpSchema";
+import db from "@/lib/db";
+import { NextRequest } from "next/server";
 
 const UsernameQuerySchema = z.object({
     username: usernameValidation
 });
 
-export async function GET(request: Request) {
-    await dbConnect();
+export const dynamic = "force-dynamic";
+
+export async function GET(request: NextRequest) {
+
 
     try {
         const { searchParams } = new URL(request.url);
         const queryParams = {
             username: searchParams.get("username")
         };
-        
+
         const result = UsernameQuerySchema.safeParse(queryParams);
-        
+
         if (!result.success) {
             const usernameErrors = result.error.format().username?._errors;
             return new Response(JSON.stringify({
                 success: false,
                 message: `${usernameErrors}`
             }), { status: 400 });
```

Comment:
Make two changes for me 

Make status 200 here

Reply:
![image](https://github.com/user-attachments/assets/15cb48a8-3e1a-40c2-a492-809d0a4cf68a)

and also here